components/mpas-ocean/src/shared/mpas_ocn_diagnostics_variables.F

```diff
@@ -72,7 +72,11 @@ module ocn_diagnostics_variables
 
    real (kind=RKIND), dimension(:), pointer :: gradSSH
    real (kind=RKIND), dimension(:), pointer :: pressureAdjustedSSH
-   real (kind=RKIND), dimension(:), pointer :: ssh_sal
+   real (kind=RKIND), dimension(:), pointer :: pgf_sal
+   real (kind=RKIND), dimension(:), pointer :: temp_twd
+   real (kind=RKIND), dimension(:), pointer :: normalCoeff 
```

Comment:
Perhaps `normalCoeffTWD`, etc to make it clear what these coefficients are for?

Reply:
I agree, thanks!